### PR TITLE
Update the game over message.

### PR DESCRIPTION
**Pull request by mossbaby**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by jt-wrk — COMMENTED** [review]
Looks good

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
```
Comment (jt-wrk) [inline index.html]: JTCW's suggestion here
```suggestion
blah blah
```

File: index.html [hunk 2]
```diff
@@ -46,8 +46,8 @@ <h1>Tic Tac Toe</h1>
     <td><input type="checkbox" /></td>
   </tr>
 </table>
-
-<h2 hidden>Game over</h2>
+blah blah
```
Comment (jtCWcom) [inline index.html:49]: remove 2nd blah

Comment (jt-wrk) [inline index.html]:
```suggestion remove 2nd blah
blah 
```

**Pull request merged**